include/seqan3/alignment/pairwise/alignment_algorithm.hpp

```diff
@@ -173,31 +175,31 @@ class alignment_algorithm :
      * |space (begin positions) |\f$ O(n*m) \f$    |\f$ O(n*k) \f$     |
      * |space (alignment)       |\f$ O(n*m) \f$    |\f$ O(n*k) \f$     |
      */
-    template <indexed_sequence_pair_range indexed_sequence_pairs_t>
+    template <indexed_sequence_pair_range indexed_sequence_pairs_t,
+              typename callback_t>
     //!\cond
-        requires !traits_t::is_vectorised
+        requires !traits_t::is_vectorised &&
+                 is_type_specialisation_of_v<
```

Comment:
shouldn't this (also) require "invokable on result type"?

Reply:
that will be anyway changed in a follow up PR since we pass now the result type as part of the alignment configuration and then can actually use the regular std::invocable concept.